### DISCUSSION TITLE
Is mayaUSD 0.15.0 still compiled with AR1.0 ?

### DISCUSSION THREAD

**Discussion by raphaeljnwave** [Autodesk/maya-usd#1968] [Q&A] [answered]
Hello mayaUSD team,

I was glad to see that mayaUSD 0.15.0 was compiled with USD 21.11 which uses the AR API 2.0 by default. But is it possible that mayaUSD 0.15.0 is still using the old AR API ? I so, why is that ? to have the new AR API, do I have to recompile mayaUSD with the AR2.0 enabled ?

Thank you !
Raphael

**Comment by seando-adsk** [comment, marked as answer]
Hi @raphaeljnwave, yes the USD v21.11 version that comes with MayaUsd v0.15.0 was compiled with AR v1. During our initial testing of USD v21.11 we discovered a problem on the **Windows** platform where some of our MayaUsd tests were failing. Upon discussion with Pixar it was found that the format of the resolved path on Windows changed with Ar 2.0.

So if you are on Windows I recommend you stay with AR v1 until the problem is fixed (either in USD or MayaUsd or both). If you are on Linux/Mac, then in order to use AR v2 you would need to build both USD and MayaUsd (using your local USD build).

Sean

**Reply by raphaeljnwave** [reply]
Thank you very much @seando-adsk  !
